_posts/en/newsletters/2022-07-06-newsletter.md

```diff
@@ -0,0 +1,192 @@
+  to block chain analysis organizations that Bob wants to pay Alice.
+
+    Hodler's scheme would be less likely to leak this information but it
+    would increase the amount of data a client implementing the protocol
+    would need to learn from the block chain, making it less suitable
+    for light clients.  Ruben Somsen suggested several alternatives
+    that could also be investigated, including Somsen's silent
+    payments idea (see [Newsletter #194][news194 silent payments]),
+    Robin Linus's [2022 stealth addresses][] idea, and [previous
+    discussion][prauge bip47] posted to the mailing list about improving
+    upon BIP47.
+
+- **Announcing splices:** in a [PR][bolts #1004] and a
+  [discussion][osuntokun splice] on the Lightning-Dev mailing list,
+  developers discussed the best way to communicate that a channel that
+  was seemingly being closed onchain was in fact a [splice][topic
+  splicing] where funds were being added to the channel or removed from
+  it.
+
+    One proposal was for nodes to simply not consider a channel closed
+    until some amount of time after its onchain closing transaction was
+    seen.  This would give time for he announcement of the new
+    (post-splice) channel to propagate.  In the interim, nodes would
+    still attempt to route payments through the seemingly-closed
+    channel, as a spliced channel would still be able to forward
+    payments with full LN security even before its new channel open
+    transaction received a suitable number of confirmations.
+
+    Another proposal was to include a signal onchain as part of the
+    closing transaction that a splice was in progress, telling nodes
+    that they could continue trying to forward payments through it.
+
+    The discussion had not come to a clear conclusion at the time this
+    summary was being written.
+
+- **Fundamental fee-collection strategies for LN forwarding nodes:**
+  developer ZmnSCPxj [summarized][zmnscpxj forwarding] three strategies
+  LN forwarding nodes can use in collecting fees for routing payments
+  (including the strategy of not collecting fees).  ZmnSCPxj then
+  analyzes possible outcomes of the different strategies.  This seems
+  related to his proposal for nodes to use routing fees to improve
+  payment success rates, see [Newsletter #204][news204 fee signal],
+  which also received significant [additional commentary][towns fee
+  signal] in the past week from Anthony Towns.
+
+- **Onion message rate limiting:** Bastian Teinturier
+  [posted][teinturier rate limit] a summary of an idea he attributes to
+  Rusty Russell for rate limiting [onion messages][topic onion
+  messages].  The proposal has each node store just an extra 32 bytes of
+  information for each of their peers that allows them to
+  probabalistically punish peers who send too much traffic.  The
+  suggested penalty is to just half the rate limit for a peer relaying
```

Comment:
(I think we're supposed to use "halve" as the verb form, but very sorry if I'm just nitpicking)
```suggestion
  suggested penalty is to just halve the rate limit for a peer relaying
```